modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -711,13 +711,15 @@ class CV_EXPORTS ImageCollection {
         int m_curr;
     };
 
-    ImageCollection();
-    ImageCollection(const String& filename, int flags);
-    void init(const String& img, int flags);
-    size_t size() const;
-    const Mat& at(int index);
+    CV_WRAP ImageCollection();
+    CV_WRAP ImageCollection(const String& filename, int flags = IMREAD_UNCHANGED);
+    CV_WRAP void init(const String& img, int flags);
+    CV_WRAP size_t size() const;
```

Comment:
size_t is not wrapped to Python and Java correctly. Java even does not support unsigned types. Please use int instead.

Reply:
i reverted size32()
`CV_WRAP size_t size() const;` seems working as python and java tests passed. also there is `CV_EXPORTS_W size_t imcount(const String& filename, int flags = IMREAD_ANYCOLOR);` what is your opinion.